foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchQueryStore.java

```diff
@@ -183,15 +184,19 @@ public Document get(String table, String id) throws FoxtrotException {
         }
         fxTable = tableMetadataManager.get(table);
         String lookupKey;
+        /**
```

Comment:
Remove if not needed.

Reply:
Kar de ye.

Reply:
Arre ye kar diya. Ek jagah rha gya tha vo next PR main ho jayega remove.